site/packages/aa-accounts/light-account/account.md

```diff
@@ -0,0 +1,50 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: aa-accounts • LightAccount
+  - - meta
+    - name: description
+      content: Overview of the LightAccount in aa-accounts
+  - - meta
+    - property: og:description
+      content: Overview of the LightAccount in aa-accounts
+---
+
+# createLightAccount
```

Comment:
this already exists no? https://omgwinning.github.io/aa-sdk-staging/packages/aa-accounts/light-account/

Reply:
This is in packages doc

Reply:
we only have createLightAccountClient. I created createLightAccount where it contains parameters and returns. On createLightAccountClient, I referred to this doc for params for accountParams.

Reply:
maybe I should just merge with that doc.

Reply:
will merge into one